### DISCUSSION TITLE
Solis cloud uploading.. parallel

### DISCUSSION THREAD

**Discussion by kyuzumaki** [wills106/homeassistant-solax-modbus#895] [Ideas]
This integration continues to work great with TCP mod bus adapters. I've had a recent issue with my inverter and have been asked to revert to the official Solis cloud stick so they can see and monitor the errors. 

This obviously breaks all my Home Assistant integrations. I may just have to accept a bit of a bloack hole while this is fixed but it got me wondering if there is any way to run both in parallel by forwarding on the data somehow? Maybe someones done it before (my search has turned up nothing).

I tried to connect the integration to the official stick as it says it will output a TCP signal but no dice. Assuming that whatever it sends is not formatted in the same way as the EW11 output.

If its any use the firmware on the stick is 
Firmware version
MW3_15_0501_1.18

I believe these are generic sticks used across various inverters brands.

**Comment by wills106** [comment]
You can use them in Parallel.

There is a link in https://homeassistant-solax-modbus.readthedocs.io/en/latest/modbus-adaptor-setup/#solis-inverters to https://github.com/alienatedsec/solis-ha-modbus-cloud

You will need some more equipment though.
